src/formatDate.js

```diff
@@ -49,8 +49,51 @@
  * @returns {string}
  */
 
+function getFormatIndexes(array, object) {
+  array.forEach((value, index) => {
+    if (index <= 2) {
```

Comment:
The `index <= 2` condition is not clear from the code. It would be better to add a comment explaining why we are only interested in the first three elements of the array or use a meaningful constant instead of `2` to enhance code readability.